tty/usbacm/usbacm.c

```diff
@@ -192,27 +188,18 @@ static usbacm_dev_t *usbacm_getByPipe(int pipe)
 
 static usbacm_dev_t *usbacm_get(int id)
 {
-	usbacm_dev_t *tmp, *dev = NULL;
+	usbacm_dev_t *dev;
 
 	mutexLock(usbacm_common.lock);
-	tmp = usbacm_common.devices;
-	if (tmp != NULL) {
-		do {
-			if (tmp->fileId == id) {
-				dev = tmp;
-				dev->rfcnt++;
-				break;
-			}
-			tmp = tmp->next;
-		} while (tmp != usbacm_common.devices);
-	}
+	dev = lib_treeof(usbacm_dev_t, node, idtree_find(&usbacm_common.devices, id));
+	dev->rfcnt++;
 	mutexUnlock(usbacm_common.lock);
 
 	return dev;
 }
 
 
-static void usbacm_free(usbacm_dev_t *dev)
+static void _usbacm_free(usbacm_dev_t *dev)
```

Comment:
It seems that this function doesn't require mutex lock so `_` prefix can be removed.